papaparse.js

```diff
@@ -408,6 +408,9 @@
 			if (typeof str === 'undefined' || str === null)
 				return '';
 
+			if (str.constructor === Date)
```

Comment:
This comparison is faster than instanceof but won't work for subclasses of Date.  If someone is weird enough to extend Date they can modify toString() to get their desired behavior.

Reply:
I think I will prefer to not overoptimize and use instanceof and make thinks works as expected.